scripts/start

```diff
@@ -16,6 +16,17 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
+[[ -f /etc/lsb-release ]] && source /etc/lsb-release
+
+if [ "${DISTRIB_ID}"="Ubuntu" ] && [ "${DISTRIB_RELEASE}"="20.04" ]; then
+  echo "Please learn to read."
```

Comment:
Lol! I understand your frustration, but I would delete the first line.

I would add another line that includes the command to upgrade.. save the noobs like me from a Google. 😅

`sudo apt update && sudo apt dist-upgrade && sudo update-manager -d`